couch/tests/test_couchv2.py

```diff
@@ -93,12 +93,12 @@ def _assert_check(aggregator, gauges):
         for gauge in gauges["erlang_gauges"]:
             aggregator.assert_metric(gauge)
 
-    for db, dd in {"kennel": "dummy", "_replicator": "_replicator", "_users": "_auth"}.items():
```

Comment:
note: Some default DBs are not present after switching to the official couch db docker images. But it shouldn't matter that much since we are still checking for all the metrics.